frontend/encore/virtual-machine.rst

```diff
@@ -0,0 +1,106 @@
+Using Encore in a Virtual Machine
+=================================
+
+You may encounter some issues when using Encore in a virtual machine, like VirtualBox or VMWare.
+
+Fix watching issues
+-------------------
+
+When using a virtual machine, your project root directory is shared with the virtual machine with `NFS`_.
+This is really useful, but it introduces some issues with files watching.
+
+You must enable `polling`_ option to make it works:
+
+.. code-block:: javascript
+
+    // webpack.config.js
+
+    // ...
+
+    // will be applied for `encore dev --watch` and `encore dev-server` commands
+    Encore.configureWatchOptions(watchOptions => {
+        watchOptions.poll = 250; // check for changes every 250 ms
+    });
+
+Fix development server
+----------------------
+
+Configure public path
+~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+    You can skip this sub-section if your app is running on ``http://localhost``
+    and not a custom local domain-name like ``http://app.vm``.
+
+When running the development server, you will probably face the following errors in the web console:
+
+.. code-block:: text
+
+    GET http://localhost:8080/build/vendors~app.css net::ERR_CONNECTION_REFUSED
+    GET http://localhost:8080/build/runtime.js net::ERR_CONNECTION_REFUSED
+    ...
+
+If your Symfony application is running on ``http://app.vm``, you must configure the public path explicitly:
+
+.. code-block:: javascript
+
+    // webpack.config.js
+
+    // ...
+
+    if (Encore.isDevServer()) {
+        Encore
+            // the default port is "8080", you can change it with the argument "--port"
+            .setPublicPath('http://app.vm:8080/build/')
+            // public path is absolute, we must define the manifest key prefix too
+            .setManifestKeyPrefix('build/');
+    }
+
```

Comment:
Same thing here: that can be done using `--public` since it was fixed in https://github.com/symfony/webpack-encore/pull/529

Reply:
Ah nice, it's even simpler because we can simply pass `--public http://app.vm:8080` argument. :+1: